src/commands/generateToken.ts

```diff
@@ -25,6 +26,9 @@ export const builder: yargs.CommandBuilder<{}, GenerateTokenArguments> = (yargs)
     demandOption: true,
   });
   yargs.option('c', { alias: 'client', type: 'string', description: 'the name of the client', demandOption: true });
+  // https://github.com/panva/jose/blob/1a3d31c467/src/lib/secs.ts
+  // https://github.com/panva/jose/blob/1a3d31c46756d4190ccc0cb9f35fa9861808ad60/src/jwt/produce.ts
+  yargs.option('t', { alias: 'ttl', type: 'string', description: '(optional) how much ttl from date.now' });
```

Comment:
1. add to the description more information on how to specify the value.
2. no need to write optional, yargs adds it by itself to required (see image).
![image](https://user-images.githubusercontent.com/12687466/167810761-6a5e85a4-bf2d-4efe-9428-25d507a7c4d5.png)